### PR TITLE
Add esMX

### PR DESCRIPTION
**Pull request by icomputo**
Add esMX

### CODE REVIEW

File: ClassicCodex.toc [hunk 1]
```diff
@@ -1,97 +1,97 @@
...
-
-# database zhCN
-db\zhCN\items.lua
-db\zhCN\units.lua
-db\zhCN\objects.lua
-db\zhCN\quests.lua
-db\zhCN\zones.lua
-db\zhCN\professions.lua
-
-# database zhTW
```
Comment (pgarneau) [inline ClassicCodex.toc]: Why get rid of this?

Comment (icomputo) [reply ClassicCodex.toc]: sorry I think it was error

**Comment by icomputo** [comment]
Do not plan to approve the changes? Many people who play from Mexico do not work the addon, it has already been tested in wow classic.

**Comment by pgarneau** [comment]
You broke Taiwan DB imports with your PR. I request some changes a while ago

**Comment by icomputo** [comment]
oks, I send the changes again, I can only test if it works in esMX

**Pull request closed without merging**